web/app_dev.php

```diff
@@ -9,13 +9,13 @@
 
 // This check prevents access to debug front controllers that are deployed by accident to production servers.
 // Feel free to remove this, extend it, or make something more sophisticated.
-if (isset($_SERVER['HTTP_CLIENT_IP'])
-    || isset($_SERVER['HTTP_X_FORWARDED_FOR'])
-    || !(in_array(@$_SERVER['REMOTE_ADDR'], array('127.0.0.1', 'fe80::1', '::1', '192.168.71.1')) || php_sapi_name() === 'cli-server')
-) {
-    header('HTTP/1.0 403 Forbidden');
-    exit('You are not allowed to access this file. Check '.basename(__FILE__).' for more information.');
-}
+//if (isset($_SERVER['HTTP_CLIENT_IP'])
```

Comment:
Instead of commenting out these settings, consider removing them altogether

Reply:
Done, also created a issue to address this issue later on when this code is going into production.
https://www.pivotaltracker.com/story/show/164140436